scene/3d/sprite_3d.cpp

```diff
@@ -794,7 +794,7 @@ void AnimatedSprite3D::_draw() {
 
 	Point2 ofs = get_offset();
 	if (is_centered()) {
-		ofs -= s / 2;
+		ofs -= Size2(s) / 2;
 	}
```

Comment:
`Point2` seems to be `typedef Vector2 Point2;` .. so as far as I can make out the Point2 changes are not necessary here. In fact `Size2` and `Point2` appear to have identical typedefs.

Reply:
Yeah, both `Point2` and `Size2` are just aliases for `Vector2`. I'd say they're more like a conceptual hint.
Anyway, I'm not sure what "Point2 changes" you're referring to (since I've done no such changes as far as I'm concerned).

Reply:
Ah my mistake, I misread this (for some reason I thought it was using ofs) ignore me! :grinning: